generated/nidcpower/session.py

```diff
@@ -3724,6 +3724,26 @@ def initiate(self):
         return _Acquisition(self)
 
     def close(self):
+        '''close
+
+        Closes the session specified in **vi** and deallocates the resources
+        that NI-DCPower reserves. If power output is enabled when you call this
+        method, the output channels remain in their existing state and
+        continue providing power. Use the ConfigureOutputEnabled
+        method to disable power output on a per channel basis. Use the
+        reset method to disable power output on all channel(s).
+
+        **Related Topics:**
+
+        `Programming
+        States <REPLACE_DRIVER_SPECIFIC_URL_1(programmingstates)>`__
```

Comment:
?

Reply:
This is to make fixing up links in help. The original link is internal to the help file and does not work here, so I replace it with something that can be easily searched for.

This is not new, pre 1.0.